app/services/config_data_sources/json_project_data_source.rb

```diff
@@ -134,18 +133,18 @@ def job_triggers_from_hash_array(job_trigger_array: nil)
       return job_trigger_array.map do |job_trigger_hash|
         type = job_trigger_hash["type"]
 
-        # currently only supports 3 triggers
-        job_trigger = nil
-        if type == FastlaneCI::JobTrigger::TRIGGER_TYPE[:commit]
-          job_trigger = CommitJobTrigger.from_json!(job_trigger_hash)
-        elsif type == FastlaneCI::JobTrigger::TRIGGER_TYPE[:nightly]
-          job_trigger = NightlyJobTrigger.from_json!(job_trigger_hash)
-        elsif type == FastlaneCI::JobTrigger::TRIGGER_TYPE[:manual]
-          job_trigger = ManualJobTrigger.from_json!(job_trigger_hash)
+        case type
+        when FastlaneCI::JobTrigger::TRIGGER_TYPE[:commit]
```

Comment:
Should we pull this out into it's own method so we don't have to duplicate it and for readability?

Reply:
Will do 👍